app/lib/submission_builder/ty2024/states/nc/documents/d400.rb

```diff
@@ -66,6 +66,15 @@ def document
                   xml.Overpayment calculated_fields.fetch(:NCD400_LINE_28)
                 end
                 xml.RefundAmt calculated_fields.fetch(:NCD400_LINE_34)
+                xml.PaymentContact do
+                  xml.PersonName do
+                    xml.FirstName sanitize_for_xml(@submission.data_source.primary.first_name, 16) if @submission.data_source.primary.first_name.present?
+                    xml.MiddleInitial sanitize_for_xml(@submission.data_source.primary.middle_initial, 1) if @submission.data_source.primary.middle_initial.present?
+                    xml.LastName sanitize_for_xml(@submission.data_source.primary.last_name, 32) if @submission.data_source.primary.last_name.present?
+                    xml.NameSuffix @submission.data_source.primary.suffix.upcase if @submission.data_source.primary.suffix.present?
+                  end
+                  xml.USPhoneNumber @submission.data_source.direct_file_data.phone_number if @submission.data_source.direct_file_data.phone_number.present?
```

Comment:
this seems to be required, so asking a q in jira currently

Reply:
will move forward - waiting for more instruction from prod/program

Reply:
Margaret guidance to temporarily default to intake phone number (if present) to pass ATS testing. We'll have to come back to rework this piece b/c it's not an ideal case and we _still_ have a real issue for real clients.

Reply:
ok, so much back and forth -- but 
1. Courtney clarified we should always have a phone number from direct file
2. Margaret wants us to use the more recent phone number from intake, if available